src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -405,12 +381,14 @@ double SolverInterfaceImpl::advance(
   Event                    e("advance", precice::syncMode);
   utils::ScopedEventPrefix sep("advance/");
 
-  PRECICE_CHECK(_state != State::Constructed, "initialize() has to be called before advance().");
+  PRECICE_CHECK(_state != State::Constructed, "initialize() and initializeData() have to be called before advance().");
+  if (_state == State::Initialized) {
+    PRECICE_WARN("initializeData() should be called before advance(). This will become mandatory in preCICE 3.0.0");
+  }
```

Comment:
This is obsolete due to the merge, right?

```suggestion
```

Reply:
Yes. Cleaned it up :+1: